tests/features/shortnulls.feature

```diff
@@ -1,20 +1,27 @@
-Feature: shortnull
+Feature: Short Nulls
+
+  **Short Nulls**: If a reduced allele with an expression character has the same expression
+  character in it's 4 field expansion, the expression character should be propagated in the
+  reduced version of the allele.
 
   Scenario Outline:
 
     Given the allele as <Allele>
     When reducing on the <Level> level (ambiguous)
     Then the reduced allele is found to be <Redux Allele>
 
+    Examples: expression characters not propagated
+      | Allele         | Level | Redux Allele                        |
+      | DRB4*01:03N    | lgx   | DRB4*01:01                          |
+      | DRB4*01:03:01N | lgx   | DRB4*01:01                          |
+      | DRB5*01:08N    | lgx   | DRB5*01:02/DRB5*01:08               |
```

Comment:
I don't understand this line.  How does DRB5*01:08N "reduce" with lgx to DRB5*01:02/DRB5*01:08?

Reply:
The `DRB5*01:08N` is considered a  `shortnull` and it expands to the allele list and when reduced to `lgx` produces `DRB5*01:02/DRB5*01:08`

```shell
$ sqlite3 ~/.pyard/pyard-Latest.sqlite3 'select * from shortnulls where shortnull="DRB5*01:08N";'
shortnull    allele_list
-----------  -----------------------------
DRB5*01:08N  DRB5*01:08:01N/DRB5*01:08:02N
```

Reply:
That was a previous test case, not a new one.